<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateCamere</class>
 <widget class="QDialog" name="updateCamere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>184</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>184</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>184</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update Camere</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nr. locuri:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pret:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nrCameraBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="tipBaieBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="nrLocuriBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okInsert">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tip baie:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelInsert">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QComboBox" name="codCaminBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nr. camera:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pretBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cod camin:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
